pkg/schemadsl/compiler/compiler.go

```diff
@@ -107,7 +107,7 @@ func Compile(schema InputSchema, prefix ObjectPrefixOption, opts ...Option) (*Co
 		return nil, err
 	}
 
-	compiled, err := translate(translationContext{
+	compiled, err := translate(&translationContext{
```

Comment:
Making the same changes to schemadsl to support erroring on duplicate `use` expressions.